go.mod

```
@@ -1,6 +1,6 @@
 module github.com/PaddleHQ/paddle-go-sdk/v3
 
-go 1.21.0
+go 1.22.0
```

```suggestion
go 1.22
```

This will make it easier to always use the latest patch version